content/en/tracing/guide/ingestion_control_page.md

```diff
@@ -0,0 +1,112 @@
+
+## Service Table View
+
+The service table contains information by service about the ingestion configuration :
+
+Services
+: The name of each service sending traces to Datadog. The table contains root and non-root services that ingested data in the over the past one hour.
+
+Ingested Traces/s
+: Average number of traces per second ingested into Datadog for the service over the past one hour.
+
+Ingested Bytes/s
+: Average number of bytes per second ingested into Datadog for the service over the past one hour.
+
+Downstream Bytes/s
+: Average number of bytes per second ingested into Datadog for which the service is responsible of the ingestion. This includes the bytes of all descending child spans, as well as spans caught by the [Error sampler][3], the [Rare sampler][4] and the [App Analytics][5] mechanism.
+
+Traffic Breakdown
+: A detailed breakdown of the destination of every trace generated by the service. See [Traffic Breakdown](#traffic-breakdown) for more information.
+
+Ingestion Configuration
+: Shows `Automatic` if the [default head-based sampling mechanism][6] from the Agent applies. If the ingestion was configured in the tracing libraries with [user-defined rules][7], the service is marked as `Configured`. To configure ingestion for a service, read how to [change the Default Ingestion Rate](#configure-the-service-ingestion-rate) from the UI.
+
+Infrastructure
+: Hosts, containers and functions on which the service is running.
+
+Service status
+: Shows `Limited Resource` when some spans are dropped due to the datadog agent reaching CPU or RAM limits, `Legacy Setup` when some spans are ingested through the legacy [App Analytics mechanism][5], and `OK` the rest of the time.
+
+Filter the page by environment, configuration and status to view services for which you need to take an action. To reduce the global ingestion volume, sort the table by the `Downstream Bytes/s` column to view services responsible for the largest share of your ingestion.
+
+**Note:** The table is powered by the [usage metrics][8] `datadog.estimated_usage.apm.ingested_spans` and `datadog.estimated_usage.apm.ingested_bytes`. These metrics are tagged by `service`, `env` and `ingestion_reason`.
+
+### Traffic breakdown
+
+The Traffic Breakdown column breaks down the destination of all traces originating from the service. It gives you an estimation of the share of traffic that is ingested vs dropped, and for which reasons.
+
+{{< img src="tracing/trace_indexing_and_ingestion/service_traffic_breakdown.png" style="width:100%;" alt="traffic breakdown of trace ingestion" >}}
+
+The breakdown is composed of the following parts:
+
+- **Complete traces ingested** (blue): The percentage of traces that have been ingested by Datadog.
+- **Complete traces not retained** (gray): The percentage of traces that have intentionally not been forwarded to Datadog by the agent or the tracer. This can happen for one of two reasons depending on your configuration:
```

Comment:
```suggestion
- **Complete traces not retained** (gray): The percentage of traces that have intentionally not been forwarded to Datadog by the Agent or the tracer. This can happen for one of two reasons depending on your configuration:
```